src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { WorkspaceAttribute } from 'src/core/types';
+import { omit } from 'lodash';
+import * as osdTestServer from '../../../../core/test_helpers/osd_server';
+
+const testWorkspace: WorkspaceAttribute = {
+  id: 'fake_id',
+  name: 'test_workspace',
+  description: 'test_workspace_description',
+};
+
+describe('workspace service', () => {
+  let root: ReturnType<typeof osdTestServer.createRoot>;
+  let opensearchServer: osdTestServer.TestOpenSearchUtils;
+  beforeAll(async () => {
+    const { startOpenSearch, startOpenSearchDashboards } = osdTestServer.createTestServers({
+      adjustTimeout: (t: number) => jest.setTimeout(t),
+      settings: {
+        osd: {
+          workspace: {
+            enabled: true,
+          },
+        },
+      },
+    });
+    opensearchServer = await startOpenSearch();
+    const startOSDResp = await startOpenSearchDashboards();
+    root = startOSDResp.root;
+  }, 30000);
```

Comment:
Is there anyway to avoid the hard-coded delay?

Reply:
Sure, I believed that I copied the delay from other integration_test cases but can not find it any more. Removed that.